app/homepage.py

```diff
@@ -80,52 +162,57 @@ def meteo_graphs(display=False):
 # Content and App layout
 # The following function is used in the main.py file to instantiate the layout of the homepage
 
+# Body container
```

Comment:
Isn't this part a duplicate of what is inside the `Homepage` function? Sorry if I am missing something obvious, but is there any reason we should keep it?